package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
```

Comment:
I guess this package.json file is created mistakenly. Can you check that out and delete it if it is created unintentionally.